cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -164,8 +166,8 @@ class TemporalTest extends DocumentingTest {
               | ** Either **`-`** or **`W`** if the next component is week of the year
               | ** **`Q`** if the next component is quarter of the year
               |
-              |If the year component is prefixed with either `-` or `+` (and is separated from the next component), `Year` is allowed to contain any number of digits.
-              |For all other cases -- i.e. the year is between `0000` and `9999` (inclusive) - `Year` must have exactly four digits (the year component is interpreted as a year of the Common Era (CE)).
+              |If the year component is prefixed with either `-` or `+`, and is separated from the next component, `Year` is allowed to contain any number of digits.
```

Comment:
That is not true, there allowed range is actually [-999999999,+999999999]. The CIP does not specify that, but since we chose to implement this with `java.time.LocalDate`, we're bound by their ranges.